### DISCUSSION TITLE
4.0 Harvest tool - app:// connection?

### DISCUSSION THREAD

**Discussion by mdrum** [NASA-PDS/registry#328] [Q&A] [answered]
The new harvest configs ask for a connection string that takes the form of `app://known/cognito/first_test.xml`. Am I supposed to know what that is? Where can I find the appropriate value to use?

This is from the front of the harvest config:
```
<harvest
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="https://github.com/NASA-PDS/harvest/blob/main/src/main/resources/conf/configuration.xsd">
  <!-- Registry configuration -->
  <!-- UPDATE with your registry information -->
  <registry auth="/path/to/auth/file">app://localhost.xml</registry>```

**Comment by jordanpadams** [comment, marked as answer]
@mdrum This will information will be provided by EN when the SBN-PSI is onboarded to the upgraded Registry system. We are currently in beta testing period, and I don't think we identified SBN-PSI as one of our beta testers. If you would like to try things out, let us know and @tloubrieu-jpl can get you setup with the appropriate configurations and permissions.

**Reply by mdrum** [reply]
Hmm okay. So should I be using the old versions of Harvest and uploading that way then? Or does that no longer work?
